contracts/builders/Builders.sol

```diff
@@ -232,6 +237,7 @@ contract Builders is IBuilders, UUPSUpgradeable, OwnableUpgradeable {
         // Update pool data
         totalPoolData.distributedRewards += newPoolRewards_;
         totalPoolData.rate = currentRate_;
+        totalPoolData.totallDeposited = totalPoolData.totallDeposited + newDeposited_ - userData.deposited;
```

Comment:
Fix typo: totallDeposited -> totalDeposited

Reply:
We can assume that totalPoolData.totallDeposited == balanceOf(Builders). Do we need this field?